blog/intro-to-justfile.md

```
@@ -0,0 +1,346 @@
+itself.
+
+### Suppressing command
+
+```{=bash}
+cat << 'EOF' >> justfile
+supress_command:
+    @echo "Only this is printed"
+full_command:
+    echo "Both command and output are printed"
+EOF
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)